### PR TITLE
virustotalv3:Feature - Add get quotas action

### PR DESCRIPTION
**Pull request by ianwills-splunk**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [ x] Testing of all the changes has been performed (for bug fixes / features)
- [ x] The manual_readme_content.md has been reviewed and added / updated if needed (for bug fixes / features)
- [ x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ x] Verify all checks are passing.
- [ x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe):

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers),
  please document them in the `manual_readme_content.md`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `manual_readme_content.md` all methods (eg, OAuth) used to authenticate
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `manual_readme_content.md`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

* Adds new action "get quotas"

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Comment by dhwanis-crest** [comment]
Hi @ianwills-splunk 
In _handle_get_quotas function, at *__daily_api_ratio , I believe in ratio value should be multiplied by 100 as you have specified the column name as “% used”.
Also, it should be up to two decimal places (no need to convert it into integer) to showcase the accurate percentage for lower value.

**Comment by ianwills-splunk** [comment]
Good point. Feel free to fix it.

Cheers,
Ian

*Ian Wills*

*Senior Professional Services Consultant | *Services West

Splunk Inc.

+1 (415) 266-3707 | ***@***.***

Livermore, CO | AMER


<http://www.splunk.com/careers>

This message is intended only for the personal, confidential, and
authorized use of the recipient(s) named above. If you are not that person,
you are not authorized to review, use, copy, forward, distribute or
otherwise disclose the information contained in the message.

**Comment by hardik-crest** [comment]
Hello @ianwills-splunk ,

There is an ongoing bug fix, which is completed. We are including those changes in your PR to release it in a single release.

File: virustotalv3_connector.py [hunk 1]
```diff
@@ -740,24 +770,23 @@ def _handle_detonate_url(self, param):
...
-
-                    try:
-                        scan_id = json_resp['data']['id']
-                    except KeyError:
-                        return self.virustotalv3_action_result.set_status(phantom.APP_ERROR, 'Malformed response object, missing scan_id.')
-                    return self._poll_for_result(self.virustotalv3_action_result, scan_id, self._poll_interval, wait_time)
-
             return self.virustotalv3_action_result.get_status()
 
+        if json_resp.get('error', {}).get('code') == 'NotFoundError':
```
Comment (alexa-phantom) [inline virustotalv3_connector.py:775]: There is 
PASS_ERROR_CODE = {
    400: 'NotAvailableYet',
    404: 'NotFoundError',
    409: 'AlreadyExistsError'
} 
available to use. Please update similar occurrences.

**Review by alexa-phantom — CHANGES_REQUESTED** [review]

File: virustotalv3_connector.py (hunk 1 above)
Comment (hardik-crest) [reply virustotalv3_connector.py]: Done

**Review by alexa-phantom — APPROVED** [review]

**Review by bbielinski-splunk — APPROVED** [review]

**Pull request merged**